ports/nlohmann-json/vcpkg.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "nlohmann-json",
+  "version-string": "3.10.0",
+  "description": "JSON for Modern C++",
+  "homepage": "https://github.com/nlohmann/json",
+  "dependencies": [
+    "vcpkg-cmake-config"
```

```suggestion
{
    "name": "vcpkg-cmake",
    "host": true
},
{
   "name": "vcpkg-cmake-config",
   "host": true
    }
```
We should add the dependencies like this for `vcpkg-cmake `and `vcpkg-cmake-config`.